draft-bbs-signatures.md

```diff
@@ -534,45 +534,47 @@ Procedure:
 
 6. generators =  (H_s || H_d || H_1 || ... || H_L)
 
-7. domain = OS2IP(HASH(PK || L || generators || Ciphersuite_ID || header)) mod q
+7. revealed_msgs = (msg_i1 || msg_i2 || ... || msg_iR)
```

Comment:
On further thought with ProofGen, where is (msg_i1, .., msg_iR) defined?

Reply:
We need to add a step to this procedure that computes msg_i1 ... msg_iR from msg and RevealedIndexes